src/map/status.c

```diff
@@ -5953,8 +5967,10 @@ short status_calc_aspd_rate(struct block_list *bl, struct status_change *sc, int
 		aspd_rate += sc->data[SC_BATTLESCROLL]->val1 * 10;
 	if (sc->data[SC_STEAMPACK])
 		aspd_rate += sc->data[SC_STEAMPACK]->val2 * 10;
+	if (sc->data[SC_SKF_ASPD])
+		aspd_rate += sc->data[SC_SKF_ASPD]->val1 * 10;
```

Comment:
+ means lower the aspd
- means increase the aspd
What's the behavior of this status?
I guess it should increase?